README.md

```diff
@@ -11,57 +11,58 @@ on https://api.judge0.com.
 2. [Project Organization](#project-organization)
 3. [Quick Development Setup](#quick-development-setup)
 4. [Quick Production Setup](#quick-production-setup)
-5. [About Docker Images](#about-docker-images)
-6. [Adding New Compiler or Interpreter](#adding-new-compiler-or-interpreter)
-7. [HTTPS In Production](#https-in-production)
-8. [Notes](#notes)
+5. [Carefully in Windows](#carefully-in-windows)
```

Comment:
I would add this section as an subsection of `Notes`. Also I would just name this `Running on Windows`.

Reply:
Maybe someone will not notice it